src/components/ChattingScreen.js

```diff
@@ -18,14 +21,96 @@ export default function ChattingScreen() {
 		{ user: false, content: '나도 아니까  ' },
 		{ user: false, content: '그만 말해줘도 돼' },
 		{ user: true, content: 'ㅠㅠ' },
-	];
+	]);
+	function changeSender(e){
+		e.preventDefault();
+		setCurrentSender(!currentSender);
+	}
+	function changeNewMsg(e){
+		e.preventDefault();
+		setNewMsg({user: currentSender, content: e.target.value});
+	}
+	function updateMsglist(e){
+		e.preventDefault();
+		if(newMsg.content === ''){
+			alert('메시지를 입력해주세요');
+			return;
+		}
+		setCurrentMsglist([...currentMsglist, newMsg]);
+		setNewMsg({user: currentSender, content: ''});
+	}
```

Comment:
여기서도 `updateMsgList(e)`라는 함수가 어떤 이벤트로 인해서 실행되는 함수인지 모르니까 이름에 그 의미를 담았으면 좋겠어요! (일반적으로 리액트에서 핸들러는 handle로 시작합니다.)
"저는 전송 버튼 눌림 이벤트를 핸들링합니다"

이벤트 핸들러 안에서 하는 일들의 추상화가 필요하다면(여기서는 sendMessage()) 내부로 코드들을 추상화해볼 수 있겠죠? 이렇게 하면 "전송 버튼을 눌렀을 때, 메세지를 전송한다"라고 추상화해놓고 그 내부의 구체적인 로직은 sendMessage() 속에 숨겨놓을 수 있습니다!

```suggestion
	function handleSendButtonClick(e) {
		e.preventDefault();
		sendMessage();
	}
	
	function sendMessage() {
                  if(newMsg.content === ''"){
			alert('메시지를 입력해주세요');
			return;
		}
		setCurrentMsglist([...currentMsglist, newMsg]);
		setNewMsg({user: currentSender, content: ''});
	}
```

Reply:
와.. 👍 sendMessage() 안에 내부 로직을 숨겨놓으니 코드가 무슨 의미인지도 잘 알겠고 가독성도 더 좋아보여요! 정말 좋은 조언 감사합니다